### PR TITLE
fix(client):improve getter

### PR DESCRIPTION
**Pull request by konata33**
There should be no delete icon in the getters function

Before:
before

After
after

### CODE REVIEW

**Comment by webfansplz** [comment]
Can we find a better way to handle it? e.g. mark pinia getters as not editable. /cc @alexzhang1030

File: packages/client/src/components/inspector/InspectorDataField/Actions.vue [hunk 1]
```diff
@@ -118,7 +119,7 @@ function quickEditNum(v: number | string, offset: 1 | -1) {
       </template>
     </template>
     <!-- delete prop, only appear if depth > 0 -->
-    <VueButton v-if="!props.disableEdit && depth > 0" v-bind="iconButtonProps" :class="buttonClass" @click.stop="quickEdit(rawValue, true)">
+    <VueButton v-if="!props.disableEdit && depth > 0 && props.rootId !== 'getters'" v-bind="iconButtonProps" :class="buttonClass" @click.stop="quickEdit(rawValue, true)">
```
Comment (Azurewarth0920) [inline packages/client/src/components/inspector/InspectorDataField/Actions.vue:122]: `rootId: 'getters'` is not exclusively used by Pinia.

You can approach like passing `{ disableEdit: true }` from props in Pinia's getter field.

**Comment by konata33** [comment]
> Can we find a better way to handle it? e.g. mark pinia getters as not editable. /cc @alexzhang1030
This problem is consistent with the cause of #231, it would be great if there were variables that could distinguish them

**Comment by alexzhang1030** [comment]
I think we can merge it for a temporal fix. For some reason, `devtools-next` is not fully compatible with `pinia` tab (due to pinia has already implemented the old `devtools` protocol`, but `devtools-next` is not.)

In the future, we will rewrite state editor protocol to be fully compatible with devtools-old.

File: packages/client/src/components/inspector/InspectorDataField/Actions.vue (hunk 1 above)
Comment (konata33) [reply packages/client/src/components/inspector/InspectorDataField/Actions.vue]: I have improved the code and thank you very much for your help. If you could help me check again, I would greatly appreciate it
@Azurewarth0920

Comment (Azurewarth0920) [reply packages/client/src/components/inspector/InspectorDataField/Actions.vue]: @konata33 
I think you can pass the `disable-edit` to component `InspectorState` (https://github.com/vuejs/devtools-next/blob/main/packages/client/src/pages/pinia.vue#L77) in getter field.

**Comment by webfansplz** [comment]
Thanks for the PR. I think we need to find a better way to handle it. Close it for now because we've made some changes to the codebase.

**Pull request closed without merging**